### ISSUE TITLE
How to block after start framework instead of getchar in simple example

### ISSUE THREAD

**Issue by chraja1233** [babelouest/ulfius#103]
Could you please tell me how to wait in simple example instead of getchar . Because my requirement is act as http server but there is no blocking call. Right now I am using infinite loop with sleep(1). Provide solution without getchar or sleep.

**Comment by babelouest** [comment]
Hello,

This is not related to Ulfius itself and you have several ways to do so. So the answer depends on the kind of program you're doing.

For example, in [Glewlwyd](https://github.com/babelouest/glewlwyd), I use a pthread condition. The program starts, then [waits](https://github.com/babelouest/glewlwyd/blob/master/src/glewlwyd.c#L256) on the condition to be signaled, and I intercept [signal calls](https://github.com/babelouest/glewlwyd/blob/master/src/glewlwyd.c#L110) to properly end the program.

So in conclusion, Ulfius won't answer your question, but C and system calls documentation will help you better.
